pkg/virt-handler/vm.go

```diff
@@ -703,11 +723,33 @@ func (d *VirtualMachineController) migrationTargetUpdateVMIStatus(vmi *v1.Virtua
 		return nil
 	}
 
+	domainExists := domain != nil
+
 	// Handle post migration
 	if domainExists && vmi.Status.MigrationState != nil && !vmi.Status.MigrationState.TargetNodeDomainDetected {
 		// record that we've see the domain populated on the target's node
 		log.Log.Object(vmi).Info("The target node received the migrated domain")
 		vmiCopy.Status.MigrationState.TargetNodeDomainDetected = true
+
+		// adjust QEMU process memlock limits in order to enable old virt-launcher pod's to
+		// perform hotplug host-devices on post migration.
+		if err := isolation.AdjustQemuProcessMemoryLimits(d.podIsolationDetector, vmi, d.clusterConfig.GetConfig().AdditionalGuestMemoryOverheadRatio); err != nil {
+			d.recorder.Event(vmi, k8sv1.EventTypeWarning, err.Error(), "Failed to update target node qemu memory limits during live migration")
```

Comment:
Now when I look at this, will we retry this error? Not really related to your PR.

Reply:
nope